doc/README.md

```diff
@@ -1,4 +1,60 @@
 KOS_DOC
 =======
+The documents are generated using Sphinx restructured text, with the ReadTheDocs
+theme.
 
-New documentation source for Sphinx/reStructuredText format.
+#Getting started on Windows
+
+(For this example, the KOS repository is assumed to be located at `C:\KOS`,
+you should adjust your path based on your actual repository location)
+
+1. If you don't already have Python installed, install the latest version in the
+  2.7 series.  At the time of this writing, 2.7.11 was the most current version.
+2. You can verify the installation and version of Python (and pip) by issuing
+  the following commands from the command line:
+  ```
+  C:\>python --version
+  Python 2.7.11
+  C:\>pip --version
+  pip
+  ```
+3. Install the Sphinx engine
+  ```
+  C:\>pip install sphinx
+  ```
+4. Ensure that the read the docs template is installed.
+  ```
+  C:\>pip install sphinx_rtd_theme
+  ```
+5. Switch to the docs directory and run the make batch file:
+  ```
+  C:\>cd KOS\docs
+  C:\KOS\docs>make html
```

Comment:
I'd add the `make clean` beforehand just to be sure.  Sure the first time there's no difference, but after that sometimes you make a change that affects more than just the source file that was itself touched.  For example, if a file has a tag :ref: in it, then it needs to be recompiled when the target of that tag moves.  Even if the file pointing to it itself didn't change, the html built from it needs a new URL placed there.